pubsublite-spark-sql-streaming/pom.xml

```diff
@@ -0,0 +1,251 @@
+    <outputDirectory>target/classes</outputDirectory>
+    <testOutputDirectory>target/test-classes</testOutputDirectory>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.coveo</groupId>
+          <artifactId>fmt-maven-plugin</artifactId>
+          <version>2.9</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>format</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <version>3.1.2</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.8.1</version>
+          <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+            <encoding>UTF-8</encoding>
+            <compilerArgument>-Xlint:unchecked</compilerArgument>
+            <annotationProcessorPaths>
+              <path>
+                <groupId>com.google.auto.value</groupId>
+                <artifactId>auto-value</artifactId>
+              </path>
+              <path>
+                <groupId>com.google.auto.service</groupId>
+                <artifactId>auto-service-annotations</artifactId>
+              </path>
+            </annotationProcessorPaths>
+          </configuration>
+        </plugin>
+
+      </plugins>
+    </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>com.coveo</groupId>
+        <artifactId>fmt-maven-plugin</artifactId>
+        <version>2.9</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>format</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M3</version>
+        <executions>
+          <execution>
+            <id>enforce</id>
+            <configuration>
+              <rules>
+                <requireUpperBoundDeps>
+                  <level>WARN</level>
+                  <excludes>
+                    <exclude>org.checkerframework:checker-compat-qual</exclude>
+                  </excludes>
+                </requireUpperBoundDeps>
+                <banDuplicateClasses>
+                  <ignoreClasses>
+                    <ignoreClass>org.apache.commons.logging.*</ignoreClass>
+                    <ignoreClass>org.apache.commons.collections.*</ignoreClass>
+                    <ignoreClass>org.apache.spark.unused.*</ignoreClass>
+                    <ignoreClass>org.apache.hadoop.yarn.*</ignoreClass>
+                    <ignoreClass>javax.ws.rs.*</ignoreClass>
+                  </ignoreClasses>
+                  <ignoreWhenIdentical>true</ignoreWhenIdentical>
```

Comment:
Let me pull this out of this PR for now. Since now it works locally, I am trying to tidying up the deps and maybe when I find a better way to package those deps it will disappear. I added those because maven failed compilation with classes duplicate.

Reply:
Unfortunately it would block this PR https://screenshot.googleplex.com/C8VCsUBqbq3KtGU. I am not aware of a better way, so I am putting this back.

Reply:
I didn't ask you to remove it, just comment and document why it is needed in the xml. Please do so.